codespell_lib/data/dictionary.txt

```diff
@@ -30105,6 +30108,7 @@ standarized->standardized
 standarizes->standardizes
 standarizing->standardizing
 standars->standards
+standars-version->standard-version, standards-version,
```

Comment:
Just confused for a moment.... so codespell won't pick up that the string `standars` within `standars-version` is a typo unless we add the `-version` to it ?

Reply:
Updated.